src/action-inputs.ts

```diff
@@ -8,7 +8,7 @@ export const getInputs = (): IActionInputs => {
   const BRANCH_IGNORE_PATTERN: string = core.getInput('skip-branches', { required: false }) || '';
   const CUSTOM_ISSUE_NUMBER_REGEXP = core.getInput('custom-issue-number-regexp', { required: false });
   const JIRA_PROJECT_KEY = core.getInput('jira-project-key', { required: false });
-
+  const FAIL_WHEN_JIRA_ISSUE_NOT_FOUND = Boolean(core.getInput('fail-when-jira-issue-not-found', { required: false }) === 'true') || false;
```

Comment:
no Boolean casting needed here, 
const FAIL_WHEN_JIRA_ISSUE_NOT_FOUND = `core.getInput('fail-when-jira-issue-not-found', { required: false }) === 'true'
` is enough
:)

Reply:
Done. It works! Thanks.